assets/js/src/register.js

```diff
@@ -103,18 +110,25 @@ jQuery( document ).ready( function($){
 		$('#error').removeClass('alert alert-danger');
 
 		$.ajax({
-			url: 'http://hospital.test/ajax/get_data',
+			url: 'http://hospital.test/ajax/register',
 			method: 'POST',
 			data: {
 				name: name,
 				email: email,
 				password: password,
 				date: date,
+				mobile: mobile,
 				gender: gender,
 				address: address
 			},
 			success: function( response ) {
 				console.log( response );
+				$('#error').html(response);
+				$('#error').addClass('alert alert-danger');
+
+				$('html, body').animate({
+					scrollTop: $('#error').offset().top
+				});
```

Comment:
- Why is there `console.log`?
- Why it's always considering as error? What if the user is successfully registered?